command/sub8_launch/launch/tf.launch

```diff
@@ -29,6 +29,10 @@
 
     <node pkg="sub8_launch" type="tf_republisher.py" name="tf_replublisher" />
 
+    <node pkg="tf" type="static_transform_publisher" name="blueview_broadcaster"
+      args="0.333 0 0.381  0 0 0 1 base_link blueview 100">
```

Comment:
I think the blueview orientation is <0 0 -0.707 0.707>, where the x-axis is pointing right, y-axis is forward, and z-axis is up.